lib/streamlit/elements/multiselect.py

```diff
@@ -237,7 +216,7 @@ def _multiselect(
         *,  # keyword-only arguments:
         disabled: bool = False,
         ctx: Optional[ScriptRunContext] = None,
-    ) -> List[Any]:
+    ) -> Union[List[V_co]]:
```

Comment:
Is the `Union` here an artifact from changes made to support the enum serialization weirdness that ended up being reverted? (same comment in a few places above)

Reply:
I think so! Good catch!

Reply:
I removed the union.